addons/stock_account/static/src/stock_account_forecasted/forecasted_header.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template">
+    <t name="stock_account.ForecastedHeader" owl="1" t-inherit="stock.ForecastedHeader" t-inherit-mode="extension">
+        <xpath expr="//h6[@name='product_variants']" position="after">
+            <h6 t-if="() => env.user.has_group('stock.group_stock_manager')">
+                Value On Hand:
+                <a href="#" class="o_report_open_valuation_report"
```

Comment:
nitpick: I think this classes are from the old code but they don't appear to have a purpose anymore. Perhaps we can delete them?